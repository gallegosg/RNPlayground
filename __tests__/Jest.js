import React from 'react';
import Jest from '../src/Jest';

import renderer from 'react-test-renderer';

test('renders correctyl', () => {
  const jest = renderer.create(<Jest />).toJSON();
  expect(jest).toMatchSnapshot();
})