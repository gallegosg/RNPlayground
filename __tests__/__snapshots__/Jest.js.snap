// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctyl 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#abbaab",
      "flex": 1,
    }
  }
>
  <Text>
     Jest 
  </Text>
</View>
`;
